README.md

```diff
@@ -12,14 +12,27 @@ Check out [Admiralty's blog post](https://admiralty.io/blog/running-argo-workflo
 
 ## Getting Started
 
-We assume that you are a cluster admin for two clusters, associated with, e.g., the contexts "cluster1" and "cluster2" in your kubeconfig. We're going to install multicluster-scheduler in both clusters, and configure cluster1 as a source and target, and cluster2 as a target only. This topology is typical of a cloud bursting use case. Then, we will deploy a multi-cluster NGINX.
+The first thing to understand is that there are two distinct kubernetes cluster types involved:
+
+1. A source kubernetes cluster
+1. One or more target kubernetes clusters
+
+Admiralty has to be installed on both types of clusters for the federation to work.
+
+Note that if a single person manages both types of clusters, that is great, but Admiralty can also be used to join clusters operated by several distinct administrattive groups.
+
+For this document we assume that you are a cluster admin for two clusters, associated with, e.g., the contexts "cluster1" and "cluster2" in your [kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/). We're going to install multicluster-scheduler in both clusters, and configure cluster1 as a source and target, and cluster2 as a target only. This topology is typical of a cloud bursting use case. Then, we will deploy a multi-cluster NGINX.
+
 
 ```bash
 CLUSTER1=cluster1 # change me
 CLUSTER2=cluster2 # change me
 ```
 
-Note: you can easily create two clusters on your machine with [kind](https://kind.sigs.k8s.io/).
+If you can only access one of the two clusters, just follow the intructions relevant to your cluster. In that case you can also remove the context part from all of the commands. Note that some parts need coordination between the admins of the two clusters; how messages are exchange in multi-admin setups is beyond the scope of this document.
+
+
+Note: you can easily create two clusters on your machine with [kind](https://kind.sigs.k8s.io/). For larger, more realistic clusters you may want to consider using [kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
```

Comment:
I added kubeadm because it has really easy to use copy-and-paste instructions.
k8s documentation is full of "fluffware" pages that lead you nowhere.
Anyway, not insisting on this; if you do not like it, I am fine with removing it.
It is indeed not Admiralty specific.

Reply:
This is fine.